google-cloud-spanner/acceptance/spanner_helper.rb

```diff
@@ -246,16 +246,20 @@ def default_item_rows
 fixture = Object.new
 fixture.extend Acceptance::SpannerTest::Fixtures
 
-job = $spanner.create_instance $spanner_prefix, name: $spanner_prefix, config: "regional-us-central1", nodes: 1
-job.wait_until_done!
-fail GRPC::BadStatus.new(job.error.code, job.error.message) if job.error?
+instance = $spanner.instance "google-cloud-ruby-tests"
+instance ||= begin
+  inst_job = $spanner.create_instance "google-cloud-ruby-tests", name: "google-cloud-ruby-tests", config: "regional-us-central1", nodes: 1
```

Comment:
What do you think about optionally loading the config from an env variable, with `regional-us-central1` as default if no env var is present?

Reply:
I don't think that is necessary. The CI environments and our development environments don't need that ability. Same as we don't need the ability to change the instance name. I think these are good/sane defaults.